rbd/rbd.go

```diff
@@ -994,21 +1004,21 @@ func GetTrashList(ioctx *rados.IOContext) ([]TrashInfo, error) {
 
 // TrashRemove permanently deletes the trashed RBD with the specified id.
 func TrashRemove(ioctx *rados.IOContext, id string, force bool) error {
-	c_id := C.CString(id)
-	defer C.free(unsafe.Pointer(c_id))
+	cid := C.CString(id)
```

Comment:
You promised you would use `cID` next time! 😭 
(just kidding, actually this time it didn't bother me anymore. 🤷‍♂️ )

Reply:
Did I? I must have forgotten.

Reply:
It must be a mind that leaks. :brain: